app/assets/javascripts/admin/components/screened-ip-address-form.js.es6

```diff
@@ -59,7 +59,7 @@ export default Ember.Component.extend({
         screenedIpAddress
           .save()
           .then(result => {
-            if (result.success) {
+            if (!result.errors) {
```

Comment:
This feels a bit odd to me at the API level. 

@nlalonde #create returns the address serialized and updater returns json success, we got to pick a side here. technically the promise will fail here if there is an error so you don't need to check for either, render_json_error defaults to 422 status

Reply:
Right, so it should be render_serialized since it calls `ScreenedIpAddress.create(result.screened_ip_address)`. But validation errors are being returned so I don't think the promise will fail, which is why we're trying to show a modal with the error.